tests/records2.yaml

```diff
@@ -37,3 +40,11 @@ record:
        type: keyword
        facets:
         facet-groups: [ 'curator' ]
+      num:
+       type: integer
+       facets:
+        facet-groups: [ 'default', 'curator' ]
```

Comment:
Should probably be updated to new facet group syntax

Reply:
Both work :)